events/open-source-marathon/index.html

```diff
@@ -466,7 +467,34 @@ <h4 class="mb-3 font-bold md:text-2xl">Concluding session and felicitation cerem
                 </div>
               </div>
             </div>
+            <!--image section starts here-->
+            
+            <div class="gallery flex flex-col lg:w-1/2 m-auto items-center">
+              <div class="overflow-y-auto">
+                  <div class="flex flex-col gap-4 sm:flex-row sm:flex-wrap md:flex-col lg:flex-row lg:flex-wrap xl:flex-col"> 
+                    <div style="display: flex;"> 
+                    <div class="w-full overflow-hidden rounded-lg shadow-lg mb-4 flex justify-center items-center">
```

Comment:
I guess this might work, but try to make it look good.

```suggestion
                    <div class="basis-1/2 grow-0 max-h-[400px] overflow-hidden rounded-lg shadow-lg mb-4 flex justify-center items-center">
```

Reply:
I am trying